mock/src/test_ctx.rs

```diff
@@ -132,22 +131,26 @@ impl<const NACC: usize, const NTX: usize> TestContext<NACC, NTX> {
             .expect("Mismatched acc len");
 
         let mut transactions = vec![MockTransaction::default(); NTX];
-        // By default, set the TxIndex and the Nonce values of the multiple transactions
-        // of the context correlative so that any Ok test passes by default.
-        // If the user decides to override these values, they'll then be set to whatever
-        // inputs were provided by the user.
-        transactions
-            .iter_mut()
-            .enumerate()
-            .skip(1)
```

Comment:
There is a `skip(1)` here. The rationale was not clear to me and in the updated implementation the first element is not skipped. But please let me know if it is necessary to skip.

Reply:
I guess it was skipping the first tx because it had a default id, which for an integer is already 0.  But I think the code looks more clear without the skip!